spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -207,6 +209,8 @@
     allow(job).to receive(:analytics).and_return(job_analytics)
     allow(IdentityConfig.store).to receive(:get_usps_proofing_results_job_reprocess_delay_minutes).
       and_return(reprocess_delay_minutes)
+    allow(IdentityConfig.store).to receive(:in_person_proofing_enforce_tmx).
+      and_return(in_person_proofing_enforce_tmx)
```

Comment:
i think we should add a test that shows `tmx_status` getting logged in a analytics event

Reply:
oh oops, I see lower down there's a convo about this

Reply:
oh wait scratch that, on re-read I think that's specifically about the `enrollment_helper` file. In that case my comment still stands.

Reply:
So I think what actually might be my preferred solution here, and doesn't conflate concerns, is to write some tests in `profile_spec.rb` that test the helper. I should have done that when pulling the logic out into a separate helper method, probably, but I overlooked it. If we trust that the helper method in profile works appropriately, and we trust that the model for profile works appropriately, and we trust that the the analytics helper logs things appropriately, how important is it to have separate testing for each thing logged in every file that implements a log event for it? 

On the other hand, maybe the answer is it is important, and I should do both. Including adding logic for `profile` to `enrollment_helper_spec`.

Reply:
Btw the analytics event does show up in the tests. They fail unless I accommodated it. The output shows `tmx_status: :threatmetrix_passed` in the `get_usps_proofing_results_job_spec` and as `tmx_status: nil` in `enrollment helper` because a profile is not defined.

Reply:
i think that makes sense. Our convention has been to add the additional property to more of the analytics tests but I do agree with you that is does add a lot to the file (perhaps unnecessarily?).

Reply:
also might be an interesting topic for eng huddle - lightening our test files by reducing some redundancy

Reply:
I'm of the opinion that if things are being tested elsewhere, then no need to duplicate the tests. I think it is good to be explicit, but if you can point to a spec that is already covering the particular thing you want to test, then maybe that's enough? But IDK, I think this warrants a broader discussion. So I second @svalexander's point above